### PR TITLE
Creating more info link and some css to home

### PR DESCRIPTION
**Pull request by LilianaRestrepoTorres**
Hello C10-Ghibli Team!
This PR solves #18 feature.

I checked the data base model and we will have the link.

- Adding link to the wiki page
- Adding css to center films on the home page
- Adding film page
- Adding goBackButton component to the film page

Screen Shot 2022-04-06 at 4 25 48 PM

### CODE REVIEW

File: src/pages/FilmView/index.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from "react";
+import { SearchEngine } from "../../components/SearchEngine";
+import { Filter } from "../../components/Filter";
```
Comment (AreYouIvan) [inline src/pages/FilmView/index.js:3]: Woops the filter css is missing

**Review by AreYouIvan — CHANGES_REQUESTED** [review]
Everything seems great, we just need to add the filter-dropdown.css or whatever it's called.. lol

File: src/pages/FilmView/index.js (hunk 1 above)
Comment (LilianaRestrepoTorres) [reply src/pages/FilmView/index.js]: Done, please check it again.

**Review by AreYouIvan — APPROVED** [review]
Cool 🚀

File: src/pages/FilmView/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    const response = await axios.get("https://ghibliapi.herokuapp.com/films");
+    setFilms(response.data);
+  }, []);
+  // Filter toggle function
+  const handleToggle = () => {
+    setToggleFilter(!toggleFilter);
+  };
+  return (
+    <>
+      <Menu />
```
Comment (Hai5edfm) [inline src/pages/FilmView/index.js:24]: if you replace the menu with the button back would be great :D, I also don't know if there should have the search component

**Review by Hai5edfm — CHANGES_REQUESTED** [review]

File: src/pages/FilmView/index.js (hunk 2 above)
Comment (LilianaRestrepoTorres) [reply src/pages/FilmView/index.js]: Done. In my opinion we should keep there search component.

**Review by Hai5edfm — APPROVED** [review]

**Pull request merged**